batch/http_to_https.go

```diff
@@ -0,0 +1,86 @@
+package batch
+
+import (
+	"database/sql"
+	"fmt"
+	"strings"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/spf13/viper"
+	"github.com/volatiletech/sqlboiler/boil"
+	"github.com/volatiletech/sqlboiler/queries/qm"
+
+	"github.com/Bnei-Baruch/mdb/models"
+	"github.com/Bnei-Baruch/mdb/utils"
+)
+
+type PostsHttpToHttps struct {
+	limit  int
+	domain string
+	mdb    boil.Executor
+	oldStr string
+	newStr string
+}
+
+func NewPostsHttpToHttps() PostsHttpToHttps {
+	domain := "youtube"
+	return PostsHttpToHttps{
+		limit:  100,
+		domain: domain,
+		oldStr: "https://www." + domain,
+		newStr: "http://www." + domain,
+	}
+
+}
+
+func (c *PostsHttpToHttps) Do() {
+	log.Infof("Start replace %s to %s for domain %s", c.oldStr, c.newStr, c.domain)
+	mdb, err := sql.Open("postgres", viper.GetString("mdb.url"))
+	utils.Must(err)
+	utils.Must(mdb.Ping())
+	defer mdb.Close()
+	c.mdb = mdb
+
+	total, err := models.BlogPosts(mdb, qm.Where(fmt.Sprintf("content ~ '.*%s*.'", c.oldStr))).Count()
```

Comment:
You don't want DB to do that. It is not indexed and not optimized.
Take all rows and filter in go code. Not in DB